README.md

```diff
@@ -64,9 +68,14 @@ defmodule VideoFilter.Pipeline do
 end
 ```
 
+Frame from original video:  
+![input](readme\input.png)  
+Output frame with filter applied:  
+![output](readme\output.png)  
```

Comment:
Invalid path separator
```suggestion
![input](readme/input.png)  
Output frame with filter applied:  
![output](readme/output.png)  
```

Also, to make the images appear on hex.pm after publish, you need to copy `readme` dir into `doc` on docs generation (see https://elixirforum.com/t/how-to-add-images-in-hex-package-documentation-with-exdoc/3103/5)